ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -352,37 +359,16 @@ Create DSCInitialization CustomResource Using Test Variables
     Run    sed -i'' -e 's/<application_namespace>/${APPLICATIONS_NAMESPACE}/' ${file_path}dsci_apply.yml
     Run    sed -i'' -e 's/<monitoring_namespace>/${MONITORING_NAMESPACE}/' ${file_path}dsci_apply.yml
     Run    sed -i'' -e 's/<operator_yaml_label>/${OPERATOR_YAML_LABEL}/' ${file_path}dsci_apply.yml
-    ${ENABLE_NEW_OBSERVABILITY_STACK} =    Get Variable Value    ${ENABLE_NEW_OBSERVABILITY_STACK}    true
-        IF    "${ENABLE_NEW_OBSERVABILITY_STACK}" == "true"
-                Run    sed -i'' -e 's/<monitoring_value>/Managed/' ${file_path}dsci_apply.yml
-        ELSE
-            IF     "${cluster_type}" == "selfmanaged"
-                    Run    sed -i'' -e 's/<monitoring_value>/Removed/' ${file_path}dsci_apply.yml
-            ELSE
-                    # If the cluster is managed, we need to set the value as Managed to keep the old monitoring stack
-                    # Once the old one is deprecated, we can remove this logic
-                    Run    sed -i'' -e 's/<monitoring_value>/Managed/' ${file_path}dsci_apply.yml
-            END
-        END
 
 Patch DSCInitialization With Monitoring Info
-    [Documentation]  Patches the DSCInitialization with the Monitoring info in case is created by the operator
+    [Documentation]  Patches the DSCInitialization with the Monitoring info if the new obs stack is being used
     ${file_path} =    Set Variable    tasks/Resources/Files/
     ${ENABLE_NEW_OBSERVABILITY_STACK} =    Get Variable Value    ${ENABLE_NEW_OBSERVABILITY_STACK}    true
         IF    "${ENABLE_NEW_OBSERVABILITY_STACK}" == "true"
```

Comment:
This check is already done before the keyword is called, we can either remove it from there or from the places where the keyword is called.